### PR TITLE
Create plugin-anka-build.yml

### PR DESCRIPTION
**Pull request by asafg6**
# Description

<a href="https://github.com/jenkinsci/anka-build-plugin">link to plugin repository</a>
<a href="https://issues.jenkins-ci.org/browse/HOSTING-371"> link to resolved hosting issue</a>

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by daniel-beck** [comment]
@asafg6 Please review the submitter checklist and follow its instructions, it makes reviewing these requests much easier. Thank you.

**Comment by asafg6** [comment]
added the links in the description

File: plugin-anka-build.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "anka-build"
+paths:
+- "org/jenkinsci/anka-build-plugin"
```
Comment (daniel-beck) [inline plugin-anka-build.yml]: Both groupId and artifactId are wrong.

File: plugin-anka-build.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+developers:
+- "asafg6"
+- "alexf123"
```
Comment (daniel-beck) [inline plugin-anka-build.yml]: Neither of these users has logged in.

**Comment by daniel-beck** [comment]
Both issues are still unresolved; please see this repository's readme for further instructions.

**Comment by asafg6** [comment]
Does my latest commit fixes the issues?

**Comment by daniel-beck** [comment]
@asafg6 

> Does my latest commit fixes the issues?

No, hence my preceding comment.

**Comment by daniel-beck** [comment]
@asafg6 In my PR review I mention specifically what's wrong, and this repository's readme explains what to do about both issues.

**Comment by asafg6** [comment]
I now have the links that was required and I changed the repository link and developers in plugin-anka-build.yml 
what am i missing here?

**Comment by daniel-beck** [comment]
> Both groupId and artifactId are wrong.

https://github.com/jenkinsci/anka-build-plugin/blob/master/pom.xml#L11...L12 doesn't look anything like https://github.com/jenkins-infra/repository-permissions-updater/pull/381/files#diff-546ce67467d14f8279bcaeb64adbb031R4

> Neither of these users has logged in.

The user has still not logged in.

Again, all of this is explained in this repository's readme, and referenced from the pull request checklist.

**Comment by asafg6** [comment]
Hey, I did the changes required. can you check it out please?

**Comment by daniel-beck** [comment]
The specified user name is now found, which means you addressed half of the issues I listed in my previous comment.

**Comment by asafg6** [comment]
ok, I changed the path as well.
looks good?

**Comment by asafg6** [comment]
oh, sorry thought it was there.

**Pull request merged**